### PR TITLE
Add User Liaison doc [SATURN-1734]

### PR DESCRIPTION
**Pull request by s-rubenstein**

### CODE REVIEW

File: roles/user-liaison.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+The user liaison is a role within the Saturn team that addresses issues that block only a few users on prod (whereas on-call addresses issues that block many users on prod) The only requirements to participate in the user liaison process are having a development setup. The user liaison is responsible for watching #saturn and #workbench-userliaison (the userliaison channel is private. If you want access for user liaison duties, message a member of the comms team)
+
+## Typical flow
+A user issue comes in through one of the channels. The user liaison jumps on to find a root cause, then either:
+* If the issue is large, make a bug ticket to bring in later, and let the reporter know you have filed a ticket
+* If the issue is small enough, file a ticket to address immediately, bring the ticket into the sprint, and let the team (and especially product owner) know
+
+## Why we have the role
+The user liaison's job is to beb a buffer against the world for the team. However, if the UL needs help, they can and should ask the team. (Their role is to handle initial triage, but in the spirit of the UL not needing to have all the answers, the team is encouraged to lift and support the UL)
```
Comment (ehanna4) [inline roles/user-liaison.md]:
```suggestion
The user liaison's job is to be a buffer against the world for the team. However, if the UL needs help, they can and should ask the team. (Their role is to handle initial triage, but in the spirit of the UL not needing to have all the answers, the team is encouraged to lift and support the UL)
```

File: roles/user-liaison.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# User Liaison
+The user liaison is a role within the Saturn team that addresses issues that block only a few users on prod (whereas on-call addresses issues that block many users on prod) The only requirements to participate in the user liaison process are having a development setup. The user liaison is responsible for watching #saturn and #workbench-userliaison (the userliaison channel is private. If you want access for user liaison duties, message a member of the comms team)
```
Comment (ehanna4) [inline roles/user-liaison.md]: This looks good but just maybe a slight re-wording about requirements being having a development setup and access to the user liaison channel:

The only requirements to participate in the user liaison process are having a development setup "and having access to the user liaison channel on slack. This channel is private so to take on user liaison duties you will need to message a member of the comms team for access. The user liaison is responsible for watching #saturn and #workbench-userliaison for any reports on bugs on our services."

**Comment by ehanna4** [comment]
Overall I think this looks great, however I would love to get somebody else's opinion like @panentheos or @breilly2 on the duties.

File: roles/user-liaison.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# User Liaison
+The user liaison is a role within the Saturn team that addresses issues that block only a few users on prod (whereas on-call addresses issues that block many users on prod) The only requirements to participate in the user liaison process are having a development setup and access to the #workbench-userliaison channel. This channel is private, so to take on user liaison duties, message a member of the comms team. The user liaison is responsible for watching #saturn and #workbench-userliaison for any reports on bugs in our services.
```
Comment (panentheos) [inline roles/user-liaison.md]: nit: Technically the role is defined across the whole workbench team and many teams participate, not just ours. Not sure if it's worth calling that out.

File: roles/user-liaison.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+## Typical flow
+A user issue comes in through one of the channels. The user liaison jumps on to find a root cause, then either:
+* If the issue is large, make a bug ticket to bring in later, and let the reporter know you have filed a ticket
+* If the issue is small enough, file a ticket to address immediately, bring the ticket into the sprint, and let the team (and especially product owner) know
+
+## Why we have the role
+The user liaison's job is to be a buffer against the world for the team. However, if the UL needs help, they can and should ask the team. (Their role is to handle initial triage, but in the spirit of the UL not needing to have all the answers, the team is encouraged to lift and support the UL)
```
Comment (panentheos) [inline roles/user-liaison.md:10]: In addition to protecting the rest of the team, the UL also provides benefit to the front-line support team as a single point of contact, who is interruptible by default (during their rotation).

**Review by panentheos — APPROVED** [review]
Looks good.

TOL: As we evolve these docs, we should keep in mind 1. who the intended audience is, 2. what they are looking to find out, and 3. what info already exists, so we don't duplicate too much.

This doc is a nice one-pager about the UL process, so we might decide we can prune some of this in favor of a link to it.

https://docs.google.com/document/d/1jFlkqccNolyHzMJpS84-PaBV4AbC3H6HKtJUmQnPeAc/edit

**Pull request merged**